Makefile

```diff
@@ -6,6 +6,7 @@ test:
 	go test -race ./...
 	go -C v2 test -race ./...
 	(cd v2 && ./hack/verify-examples.sh)
+	(cd ./examples/defaulter-gen/_output_tests && go test -v -race ./...)
```

Comment:
why not run ./hack/verify-examples.sh in the root?

Reply:
v2 has its own gomod which would change the go behavior whether this is run in the root vs in v2 directory right?

Reply:
I don't understand what this means.

hack/verify-examples.sh removes generated code (in v1 only), then runs the generators (in v1 only), then runs tests (in v1 only).  That is inclusive of 

I am suggesting this rule should basically be:

```
test: testv1 testv2

testv1:
    go test -race ./...
    ./hack/verify-examples.sh

testv2:
    go -C v2 test -race ./...
    (cd v2 && ./hack/verify-examples.sh)

Reply:
Oh, I see that's another go.mod.  Sigh.  Editing comment

Reply:
Also, keep in mind v2 is alive, v1 is in stasis. :)